### DISCUSSION TITLE
what exatcly is this objects.githubusercontent.com and why wont it let me finish the download?

### DISCUSSION THREAD

**Discussion by Superdrew907** [community/community#141907] [New to GitHub]
For some reason this domain has it out for me, started blocking my downloads I dunno how I pissed it off,.. can someone help me out here and explain this to me? I put in my username and password that doesnt work.
![image](https://github.com/user-attachments/assets/daac9ddc-f7ee-40c9-9ac0-307db5b29123)

**Comment by samuelreichor** [comment]
@Superdrew907 
The domain objects.githubusercontent.com is used by GitHub to serve large binary files (such as assets, images, or binaries) that are stored in GitHub repositories.
There could be a few reasons why you're having trouble finishing the download, but I've never had that issue:

- Network Issues: There may be a network timeout, poor connection, or an issue with the content delivery network (CDN) that GitHub uses to serve these files.
- Firewall or Security Software: Certain firewall rules, VPN configurations, or security software may block access to objects.githubusercontent.com, preventing the download from completing.
- Rate Limits: If you've been making too many requests or large downloads from GitHub, you may have hit GitHub's rate limits or quota, particularly if using Git LFS. GitHub limits the amount of data that can be downloaded or transferred in a certain time period.
- Git LFS Quota: If the repository you're working with is using Git LFS, and the repository owner has exceeded their LFS storage or bandwidth limits, the download could fail.

Hopefully this helps

**Reply by Superdrew907** [reply]
Hey, thanks sam for taking the time to respond, Im new here so I dont know shot. . I appreciate the help. my our AI Overloards bless u. take care!

**Comment by Bigsheun** [comment]
I found this site when I had the same problem. Something about the answer made me think of using tor browser. 
For the lif of me I don't really know why, but it worked
